src/Locale/es/cake.po

```diff
@@ -209,19 +212,19 @@ msgstr "Este campo es obligatorio"
 msgid "This field cannot be left empty"
 msgstr "Este campo no puede quedar vacío"
 
-#: Validation/Validator.php:1447
+#: Validation/Validator.php:1501
 msgid "The provided value is invalid"
 msgstr "Los valores proporcionados no son válidos"
 
-#: View/Helper/FormHelper.php:946
+#: View/Helper/FormHelper.php:947
 msgid "New %s"
-msgstr "Nuevo registro"
+msgstr "Nuevo %s"
```

Comment:
This "trick" was made to avoid inconsistencies in noun genders... think of "Nuevo Sección"